### ISSUE TITLE
BAM Headtracking

### ISSUE THREAD

**Issue by dacstyle** [vpinball/vpinball#1730]
I have been testing the BAM HEADTRACKING and the result is really surprising. Much better than I expected, and I am really going to play regularly in this mode. 

I have found some things that seem to be bugs to improve, such as in many tables, the ball's texture loads in a very strange way and looks bad, while in others, the flippers seem to be very tilted forward (as if we were looking at the table while crouching), but the rest of the table has a perfect perspective. The strangest case was in WhiteWater, where starting the table with Headtracking active, the flippers do not respond normally when pressing the buttons.

If it is helpful, I can share the information I am gathering through this medium to help improve these issues, or if you prefer, I can open a case for each one.

I would also like to suggest adding the possibility in the key configuration to set a key that can activate/deactivate the headtracking function, as is done with 3D (it would also be great if the 3D key could be configurable and not fixed).

**Comment by StrangeCloudsVPin** [comment]
I don’t believe my request was a duplicate. This request pertains to head tracking alone. I was hoping that you could add the functionality to use both features at the same time.
